<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>479</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>479</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>479</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../picture/logo_car.jpeg</normaloff>../picture/logo_car.jpeg</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>431</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../picture/analytics-small.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>秋名山汽车租赁系统</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>331</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="rdb_1">
      <property name="styleSheet">
       <string notr="true">*{	
	font-family:微软雅黑;
	font-size:18px;
	color: #1d649c;
}</string>
      </property>
      <property name="text">
       <string>用户</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rdb_2">
      <property name="styleSheet">
       <string notr="true">*{	
	font-family:微软雅黑;
	font-size:20px;
	color: #1d649c;
}</string>
      </property>
      <property name="text">
       <string>管理员</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>131</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="edit_username">
      <property name="placeholderText">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_password">
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="login_button">
      <property name="styleSheet">
       <string notr="true">*{	
	font-family:微软雅黑;
	font-size:15px;
	color: #1d649c;
}</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
